distribution/src/main/appended-resources/META-INF/NOTICE

```diff
@@ -58,3 +58,5 @@ This product includes code from the Nimbus JOSE + JWT project, under the Apache
 license 2.0 (https://github.com/felx/nimbus-jose-jwt). Copyright 2012-2017,
 Connect2id Ltd.
 
+Portions of this software were developed at
```

Comment:
First of all, thanks so much for the contribution!

But I'm totally not sure if we should/can add this into NOTICE. Let's wait other comments from the community.